lib/dbservice/batches/batch.ex

```diff
@@ -6,13 +6,15 @@ defmodule Dbservice.Batches.Batch do
 
   alias Dbservice.Programs.Program
   alias Dbservice.Groups.GroupType
+  alias Dbservice.Schools.EnrollmentRecord
 
   schema "batch" do
     field :name, :string
     field :contact_hours_per_week, :integer
 
     has_many :group_type, GroupType, foreign_key: :child_id, where: [type: "batch"]
     many_to_many :program, Program, join_through: "batch_program", on_replace: :delete
+    has_many :group_fk_id, EnrollmentRecord, foreign_key: :group_id
```

Comment:
Can you please help in understanding this association?

Reply:
Tagged in discord conversation.

Reply:
Replied there!